### ISSUE TITLE
[New upgrade] Disarm

### ISSUE THREAD

**Issue by Hetachi** [peace-maker/smrpg#91]
Would it be possible to create an upgrade, disarm, which disarms the enemy and yourself by striping primary and secondary weapon off of you, but has really small chance of that happening. 

Pros and Cons anyone?

**Comment by Nobody-x** [comment]
Yes it's possible, but why buy an upgrade which desarm yourself ?

**Comment by coty9090** [comment]
Cons: Very, VERY annoying RPG skill....

**Comment by Nobody-x** [comment]
This skill already exists in thcrpg

**Comment by coty9090** [comment]
I think it should be added, but it's the most annoying rpg skill i've ever seen.

**Comment by peace-maker** [comment]
What's the point of this?

**Comment by coty9090** [comment]
Config to set what weapons this upgrade works with and what weapons it can/can't drop. (Current equipped weapon only. Only applies if you get a headshot or armshot)
